libraries/AP_GPS/AP_GPS_ERB.h

```diff
@@ -66,6 +66,13 @@ class AP_GPS_ERB : public AP_GPS_Backend
         uint8_t fix_type;
         uint8_t fix_status;
         uint8_t satellites;
+        // Introduced in ERB version 0.2.0
+        int8_t numLeapSeconds; ///< leap seconds (0xFF indicates invalid)
+        int32_t baselineN__mm; ///< distance between rover and base - North - in millimeters
```

Comment:
Its a units separator. It allows one to do "inertial_speed__cm_p_s" (centimeters per second) It separates variable name from units name, because variable name can contain _ and units can also contain a _. So using a double __ makes it clear where the separation is. This example uses pretty simple units.... but there are complicated units out there. This is a standard code guideline at the company that I work.